sdk/core/azure-core/test/ut/transport_adapter_curl.cpp

```diff
@@ -8,8 +8,8 @@
 #include <string>
 #include <thread>
 
-#include "../../src/http/curl/curl_connection.hpp"
-#include "../../src/http/curl/curl_connection_pool.hpp"
+#include <http/curl/curl_connection_private.hpp>
+#include <http/curl/curl_connection_pool_private.hpp>
```

Comment:
https://azure.github.io/azure-sdk/cpp_design.html#physical-design

I think this violates our guidelines

Reply:
@Jinming-Hu  see this issue https://github.com/Azure/azure-sdk-for-cpp/issues/1043

We want to use angle brackets for tests.